examples/secrets-management-with-csi-secrets-driver-aws/data.tf

```diff
@@ -0,0 +1,11 @@
+data "aws_iam_policy_document" "secrets_management_policy" {
```

Comment:
when you move this over to the `complete-kubernetes-addons` example, no need to put this in a separate file. Instead, locate it next to where it is used. Here that would be either before or after https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/471/files#diff-6355537d0664dc024a0086c9581f3a707332dd67c563f98cbe5ee18d4fd0e44bR164